promql/promqltest/testdata/functions.test

```diff
@@ -678,10 +671,10 @@ load 10s
   metric10 -9.988465674311579e+307 9.988465674311579e+307
 
 eval instant at 1m avg_over_time(metric[1m])
-  {} 3
+  {} 3.5
 
 eval instant at 1m sum_over_time(metric[1m])/count_over_time(metric[1m])
-  {} 3
+  {} 3.5
```

Comment:
The tests intend to use all the samples stated. Let's just shift the eval time a bit to keep it that way, , i.e. `at 1m` → `at 55s`.